plugins/module_utils/identity/keycloak/keycloak.py

```diff
@@ -48,7 +48,9 @@
 
 URL_CLIENT_ROLES = "{url}/admin/realms/{realm}/clients/{id}/roles"
 URL_CLIENT_ROLE = "{url}/admin/realms/{realm}/clients/{id}/roles/{name}"
-URL_CLIENT_ROLE_COMPOSITES = "{url}/admin/realms/{realm}/clients/{id}/roles/{name}/composites"
+URL_CLIENT_ROLE_COMPOSITES = (
+    "{url}/admin/realms/{realm}/clients/{id}/roles/{name}/composites"
+)
```

Comment:
Please undo all unrelated formattings from this PR. It should only modify the lines needed, not random other ones.

Reply:
okay